### PR TITLE
AGW-4213: Review MMA documentation

### PR DESCRIPTION
**Pull request by Pgarrett**

### CODE REVIEW

**Review by balbifm — APPROVED** [review]

File: modules/ROOT/pages/migration-api-gateways-proxies.adoc [hunk 1]
```diff
@@ -73,11 +73,12 @@ xref:https://docs.mulesoft.com/exchange/to-publish-assets-maven[Publish and Depl
 === POM Migration
 
 The POM file migration modifies the file to include the elements necessary for
-uploading the custom proxy to Exchange:
+uploading the custom proxy to Exchange.
+
+After the migration:
```
Comment (dukesphere) [inline modules/ROOT/pages/migration-api-gateways-proxies.adoc]:
```suggestion
_After_ the migration:
```

File: modules/ROOT/pages/migration-api-gateways-proxies.adoc [hunk 2]
```diff
@@ -73,11 +73,12 @@ xref:https://docs.mulesoft.com/exchange/to-publish-assets-maven[Publish and Depl
...
+
+Replace the `{orgId}` value in the `<groupId/>` and `<exchange.url/>` elements with the organization ID found in Anypoint Platform.
```
Comment (dukesphere) [inline modules/ROOT/pages/migration-api-gateways-proxies.adoc]:
```suggestion
* Replace the `{orgId}` value in the `<groupId/>` and `<exchange.url/>` elements with the organization ID found in Anypoint Platform.
```

File: modules/ROOT/pages/migration-api-gateways-proxies.adoc [hunk 3]
```diff
@@ -196,8 +197,27 @@ by the Mule Migration Assistant.
 [[wsdl_proxy_migration]]
 === WSDL Proxy Migration
 
-Currently, WSDL proxy migration is not supported.
+WSDL Proxy migration consists on migrating the attribute values for WSDL properties. In Mule 3, most values where extracted using the function defined as attribute value. In Mule 4 there is no equivalent for doing so, therefore most properties are migrated to a default value and require manual configuration from the user with values taken from the WSDL file. These are the ones starting with the `service` keyword.
```
Comment (dukesphere) [inline modules/ROOT/pages/migration-api-gateways-proxies.adoc]:
```suggestion
WSDL proxy migration consists of migrating the attribute values for WSDL properties. In Mule 3, most values are extracted using the function defined as attribute value. In Mule 4, there is no equivalent process, so most properties are migrated to a default value and require manual configuration with values from the WSDL file, which start with the `service` keyword.
```

File: modules/ROOT/pages/migration-api-gateways-proxies.adoc [hunk 4]
```diff
@@ -196,8 +197,27 @@ by the Mule Migration Assistant.
...
+
+Some properties are just renamed in order to extract in a more transparent way the value, such as:
```
Comment (dukesphere) [inline modules/ROOT/pages/migration-api-gateways-proxies.adoc]:
```suggestion
Some properties are renamed to extract the value in a more transparent way, for example:
```

File: modules/ROOT/pages/migration-api-gateways-proxies.adoc [hunk 5]
```diff
@@ -196,8 +197,27 @@ by the Mule Migration Assistant.
...
+[cols="2,5",options="header"]
+|===
+|Mule 3                                  | Mule 4
 
+| `![wsdl(p['wsdl.uri']).services[0].preferredPort.name]`	| `${service.port}`
+| `![wsdl(p['wsdl.uri']).targetNamespace]` 					| `${service.namespace}`
+| `![wsdl(p['wsdl.uri']).services[0].name]`					| `${service.name}`
+|===
+
+However, other properties such as the following are migrated to a DataWeave expression which uses the WSDL Functions Extension dependency, provided by MMA, to extract the expected values:
```
Comment (dukesphere) [inline modules/ROOT/pages/migration-api-gateways-proxies.adoc]:
```suggestion
Other properties, such as the following, are migrated to a DataWeave expression that uses the WSDL Functions Extension dependency, provided by MMA, to extract the expected values:
```

**Review by dukesphere — COMMENTED** [review]
suggested edits

**Review by dukesphere — APPROVED** [review]
suggested changes were merged

**Pull request merged**